support/cas-server-support-authy/src/main/java/org/apereo/cas/adaptors/authy/config/AuthyConfiguration.java

```diff
@@ -123,10 +123,10 @@ public FlowDefinitionRegistry authyAuthenticatorFlowRegistry() {
     @Bean
     public AuthenticationHandler authyAuthenticationHandler() {
         try {
-            final AuthyAuthenticationHandler h = new AuthyAuthenticationHandler(authyClientInstance());
+            final boolean forceVerification = casProperties.getAuthn().getMfa().getAuthy().isForceVerification();
+            final AuthyAuthenticationHandler h = new AuthyAuthenticationHandler(authyClientInstance(), forceVerification);
             h.setServicesManager(servicesManager);
```

Comment:
Yeah, but that are properties from the parent class and I will do it in a separate change